lua/user/plugins.lua

```diff
@@ -58,8 +59,7 @@ return packer.startup(function(use)
   use { "goolord/alpha-nvim", commit = "ef27a59e5b4d7b1c2fe1950da3fe5b1c5f3b4c94" }
 
   -- Colorschemes
-  use { "folke/tokyonight.nvim", commit = "8223c970677e4d88c9b6b6d81bda23daf11062bb" }
-  use { "lunarvim/darkplus.nvim", commit = "2584cdeefc078351a79073322eb7f14d7fbb1835" }
+  use { "misku/monokai.nvim" }
```

Comment:
I really liked the part of adding the missing comments/descriptions on the options in this PR, but this part of changing the colorscheme I guess is not necessary. First of all I couldn't find this "misku/monokai.nvim" repo in Github and also it is missing a commit, which it kinda breaks the purpose of the repo, to have stable configs.

Reply:
Sorry, I forgot to switch branches before adding my custom changes and the commits piled up in here. It's fixed now - it's just the descriptions in the pull request.